chef_master/source/custom_resources.rst

```diff
@@ -581,10 +581,10 @@ The ``converge_if_changed`` method may be used multiple times. The following exa
    property :content, String
    property :mode, String
 
-   load_current_value do
-     if ::File.exist?(new_resource.path)
-       content IO.read(new_resource.path)
-       mode ::File.stat(new_resource.path).mode
+   load_current_value do |desired|
```

Comment:
Why not use new_resource as the block variable name?

Reply:
I ran into this situation helping some people write custom resources this week.  We had to read the resource code to figure out how load_current_value was implemented to get it to work.